lib/elixir/lib/code.ex

```diff
@@ -424,6 +424,14 @@ defmodule Code do
     end
   end
 
+  defp module_attribute_or_atom(acc, rest) do
+    case Enum.reverse(rest) do
+      '' -> {:module_attribute, acc}
+      ':' ++ atom -> {:unquoted_atom, atom ++ '@' ++ acc}
+      _ -> :none
+    end
```

Comment:
Hey, I'm pretty sure that this function isn't implemented in the best way but I was trying to get it to work. How could I improve this fix?